miner/worker.go

```diff
@@ -643,20 +643,22 @@ func (env *Work) commitTransactions(transactions types.Transactions, gasPrice *b
 			}
 		default:
 			env.tcount++
+			go mux.Post(core.PendingLogsEvent{Logs: logs})
```

Comment:
Wouldn't it be better to gather all the pending logs and only post them after exiting from the transaction iteration? Otherwise the filter load will increase with transaction count.